ovsnl/client.go

```diff
@@ -118,12 +111,10 @@ func (c *Client) initFamily(f genetlink.Family) error {
 			c: c,
 		}
 		return nil
-	case ovsh.FlowFamily, ovsh.PacketFamily, ovsh.VportFamily, ovsh.MeterFamily:
-		// TODO(mdlayher): populate.
+	default:
+		// Unknown OVS netlink family, nothing we can do.
```

Comment:
Since this function now cannot return a non-nil error, should the function signature be changed.

Reply:
Yes, I suppose it could. Thanks for the 👀

Reply:
Alternatively I can keep the current signature and continue on https://github.com/digitalocean/go-openvswitch/pull/88/files#diff-96266f493c1ce0ec7b82caccd25c17afR93, what do you reckon?